src/main/java/chzzk/grassdiary/domain/reward/RewardHistory.java

```diff
@@ -0,0 +1,26 @@
+package chzzk.grassdiary.domain.reward;
+
+import chzzk.grassdiary.domain.base.BaseCreatedTimeEntity;
+import chzzk.grassdiary.domain.member.Member;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@Entity
+public class RewardHistory extends BaseCreatedTimeEntity {
+    @Id
+    @GeneratedValue( strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    private Member member;
+    @ManyToOne
+    private RewardType rewardType;
```

Comment:
```suggestion
    @Enumerated(EnumType.STRING)
    private RewartType rewartType;
```

ref. RewartType Enum 참조

Reply:
혹시 `@ManyToOne` 부분도 enum 사용으로 `@JoinColumn`과 같이 사라지게 되는 동일한 이유인가요??

Reply:
네 맞습니다!
백엔드 회의 후, RewardType 테이블을 삭제하고 Diary 테이블의 칼럼으로 RewardType을 추가하기로 결정해서 이제 연관 관계 어노테이션을 사용하지 않고, `@Enumerated` 어노테이션을 사용해서 enum 값을 DB에 저장하도록 수정했습니다:)!

Reply:
앗 수정했는데 변경이 되지 않았군요ㅠㅠ
방금 수정했습니다ㅎㅎ